forwardproxy.go

```diff
@@ -251,7 +255,7 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request, next caddyht
 
 	if r.Method == http.MethodConnect {
 
-		if r.ProtoMajor == 2 {
+		if r.ProtoMajor == 2 || r.ProtoMajor == 3 {
```

Comment:
(same here, do we know it works with HTTP/3?)

HTTP/3 seems like a change unrelated to the Caddyfile, these might be a mistake.

Reply:
You got me. It is really unrelated and I thought it is trivial to fix here.

But I did test it and it was working more or less, though the whole QUIC pipeline for proxy is still unstable.